controllers/operatorpolicy_status.go

```diff
@@ -693,6 +700,13 @@ func installPlanUpgradeCond(versions []string, approvableIPs []unstructured.Unst
 		cond.Message += " but multiple of those match the versions specified in the policy"
 	}
 
+	if complianceConfig == "NonCompliant" {
+		cond.Status = metav1.ConditionFalse
+	} else {
+		cond.Status = metav1.ConditionTrue
+		cond.Message += " - ignore, compliance overridden by complianceConfig"
```

Comment:
I don't think this additional message is required. Now that the default is "Compliant", I don't think overridden is the correct term.

Reply:
I think because this condition is a NonCompliant condition, it would make sense to to say it was overridden. Especially since the condition message makes it to the final policy compliance event, it might be useful to see this detail.

Reply:
I think I somehow agree with both of you :laughing: - saying its overriden doesn't feel right, but I don't want to *not* mention that the possible upgrade is being ignored.

Maybe a rephrasing would help, something like " - not triggering noncompliance because of complianceConfig".